dsgrid/time/types.py

```diff
@@ -24,10 +25,30 @@ class Season(DSGEnum):
 
 # The types below represent the timestamps that exist as columns in all datasets.
 
-DatetimeTimestampType = namedtuple("DatetimeTimestampType", ["timestamp"])
 
-AnnualTimestampType = namedtuple("AnnualTimestampType", ["time_year"])
+class DatetimeTimestampType(NamedTuple):
```

Comment:
The change from namedtuple to NamedTuple allows type annotations to work on the fields.